controllers/storagecluster/uninstall_reconciler.go

```diff
@@ -293,14 +293,6 @@ func (r *StorageClusterReconciler) verifyNoStorageConsumerExist(instance *ocsv1.
 // deleteResources is the function where the storageClusterFinalizer is handled
 // Every function that is called within this function should be idempotent
 func (r *StorageClusterReconciler) deleteResources(sc *ocsv1.StorageCluster) (reconcile.Result, error) {
-
-	if IsOCSConsumerMode(sc) {
-		err := r.verifyNoStorageClassClaimsExist(sc)
```

Comment:
I think we will need a similar func to make sure we don't have the storage class requests on the provider before uninstalling. @Madhu-1 WDYT?

Reply:
@iamniting Thanks for checking, but We dont have this check for the provider cluster when storageclassclaim was present but we do have checks for storage consumer to block uninstallation, storageclassclaim/storagerequest cannot exist without storage consumer, we are good here :+1: